Firestore/core/src/remote/bloom_filter.h

```diff
@@ -51,6 +62,14 @@ class BloomFilter final {
     return bit_count_;
   }
 
+  // When checking membership of a key in bitmap, the first step is to generate
+  // a 128-bit hash, and treat it as 2 distinct 64-bit hash values, named `h1`
+  // and `h2`, interpreted as unsigned integers using 2's  complement encoding.
+  struct Hash {
```

Comment:
nit: Move `struct Hash` to the `private` section; it is not part of the public interface of this class.